builtin/common/chatcommands.lua

```diff
@@ -65,9 +56,22 @@ local function do_help_cmd(name, param)
 			end
 		end
 		table.sort(cmds)
-		return true, gettext("Available commands: ") .. table.concat(cmds, " ") .. "\n"
-				.. gettext_replace("Use '$1help <cmd>' to get more information,"
-				.. " or '$1help all' to list everything.", cmd_marker)
+		local msg
+		if INIT == "game" then
+			msg = S("Available commands: @1",
+				table.concat(cmds, " ")) .. "\n"
+				.. S("Use '@1help <cmd>' to get more "
+				.. "information, or '@2help all' to list "
+				.. "everything.",
+				cmd_marker, cmd_marker)
```

Comment:
`cmd_marker` is pointless here if `INIT` chases are handled differently. For consistency, `/help` would fit better here (without replacements).
Also inconsistent use of replacement in `Available commands: [@1]`.